### ISSUE TITLE
Add support for bit field datasets

### ISSUE THREAD

**Issue by jamesmudd** [jamesmudd/jhdf#84]
There are some examples files available at https://zenodo.org/record/1310934 currently they fail to open with jhdf because they contain bitfield attributes.

The error is
```
Caused by: io.jhdf.exceptions.HdfException: Unrecognized data class = 4
	at io.jhdf.object.datatype.DataType.readDataType(DataType.java:55)
	at io.jhdf.object.message.AttributeMessage.<init>(AttributeMessage.java:55)
```

**Comment by jamesmudd** [comment]
Some work towards this https://github.com/jamesmudd/jhdf/pull/153

**Comment by jamesmudd** [comment]
This has been done in PR https://github.com/jamesmudd/jhdf/pull/163 and is available in v0.5.5 https://github.com/jamesmudd/jhdf/releases/tag/v0.5.5
